apps/api/src/user/service/user.service.ts

```diff
@@ -51,6 +82,29 @@ export class UserService {
       isActive: dto.isActive,
       isOnboardingFinished: dto.isOnboardingFinished
     }
+
+    if (dto.email) {
+      const userwithEmail = await this.prisma.user.findFirst({
```

Comment:
Renamve this variable to `userExists` and make it boolean using `count` or `findUnique`

Reply:
Renamed